addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3011,6 +3012,37 @@ export class OdooEditor extends EventTarget {
                         );
                     }
                 }
+                if (ev.data === '`') {
+                    // We just inserted a backtick, check if there was another
+                    // one before.
+                    const range = getDeepRange(this.editable, { joinText: true, select: true });
```

Comment:
It is not clear to me why we need to add yet another option to an already convoluted function ?

Reply:
I could have just put the code right here but I figured it could be useful and would make more sense to have there. I can move it back if you prefer but I find this clearer personally.

Reply:
Well I'd argue that the main reason why it is clearer is because it hides the actual complexity into the blackhole that is `getDeepRange`... ^^' If you can think of another valid use case for which we might want to call `getDeepRange` with this option then I'm not fundamentaly opposed to it, but otherwise I think it makes more sense to have it here.

Reply:
Ok for me.

Reply:
Looks like you missed the second question... :'(

Reply:
Done.